### PR TITLE
login and registration complete.

### PR DESCRIPTION
**Pull request by pdadlani**
small bug in private route that needs to be addressed

### CODE REVIEW

File: cs-unknown/src/components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  console.log('status in Login', status)
+  console.log('user in Login', user)
+
+  useEffect(() => {
+    if (status) {
+      setUser(status);
+      console.log('useeffectttttt', history)
+      history.push('/home')
+    }
+  }, [status]);
```
Comment (danbergelt) [inline cs-unknown/src/components/Login/Login.js]: What's the purpose of this useEffect? Why not just push the user to /home in the handleSubmit?

File: cs-unknown/src/components/Registration/Registration.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        )}
+        <Field 
+          type='password'
+          name='password2'
+          placeholder='Password Confirmation'
+        />
+        {touched.password2 && errors.password2 && (
+          <p className='error'>{errors.password2}</p>
+        )}
+        <Link to='/login'>
```
Comment (danbergelt) [inline cs-unknown/src/components/Registration/Registration.js]: I don't think it's best practice to wrap the button with a link, why not just push them to login on a successful registration (like how you are doing in login)?

File: cs-unknown/src/components/Registration/Registration.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          <button type='submit'>Submit</button>
+        </Link>
+      </Form>
+      <Link to='regist'></Link>
+    </div>
```
Comment (danbergelt) [inline cs-unknown/src/components/Registration/Registration.js]: What's this?

File: cs-unknown/src/components/Registration/Registration.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      .max(12, "Do not make it super complicated."),
+    password2: Yup.string()
+      .required()
+  }),
+
+  handleSubmit(values, { setStatus }) {
+    return axios
+      .post(`https://unknown-mud.herokuapp.com/api/registration/`, values)
+      .then(res => {
+        console.log("axios response", res);
```
Comment (danbergelt) [inline cs-unknown/src/components/Registration/Registration.js]: Remember to clean up console.logs()

File: cs-unknown/src/components/Login/Login.js (hunk 1 above)
Comment (pdadlani) [reply cs-unknown/src/components/Login/Login.js]: it's also setting the user so you have the key for the logged in user. Do you have recommendations on where to set the user info so we can get rid of this?

File: cs-unknown/src/components/Registration/Registration.js (hunk 3 above)
Comment (pdadlani) [reply cs-unknown/src/components/Registration/Registration.js]: it has been updated. It was supposed to be a simple link to login page, if user already has an account.

File: cs-unknown/src/components/Registration/Registration.js (hunk 4 above)
Comment (pdadlani) [reply cs-unknown/src/components/Registration/Registration.js]: Thank you :)

**Pull request merged**